frontend/cypress/integration/login/survey-list/survey-list.spec.js

```diff
@@ -0,0 +1,33 @@
+context('survey list', ()=> {
```

Comment:
This file is under `login/survey-list`, maybe it should be only `survey-list`
The code for testing login should be under `login`